index.route.js

```diff
@@ -21,6 +21,13 @@ api.use('/teams', teamRoutes);
 api.use('/webhook', webhookRoutes);
 
 router.get('/', (req, res) => {
+  if (process.env.OPEN_FOR_REGISTRATION) {
```

Comment:
```suggestion
  if (config.isOpenForRegistrations) {
```

Reply:
@tbushman I think you need to pick it up from the config object. That will fix it.